File: app/src/main/java/com/example/zombicide/models/Skill.java
package com.example.zombicide.models;

import org.jetbrains.annotations.NotNull;

public class Skill
{
    private String skill_name_fr;

    private String description_fr;

    private String skill_name_en;

    private String skill_id;

    public String getSkill_name_fr ()
    {
        return skill_name_fr;
    }

    public void setSkill_name_fr (String skill_name_fr)
    {
        this.skill_name_fr = skill_name_fr;
    }

    public String getDescription_fr ()
    {
        return description_fr;
    }

    public void setDescription_fr (String description_fr)
    {
        this.description_fr = description_fr;
    }

    public String getSkill_name_en ()
    {
        return skill_name_en;
    }

    public void setSkill_name_en (String skill_name_en)
    {
        this.skill_name_en = skill_name_en;
    }

    public String getSkill_id ()
    {
        return skill_id;
    }

    public void setSkill_id (String skill_id)
    {
        this.skill_id = skill_id;
    }

    @NotNull
    @Override
    public String toString()
    {
        return "ClassPojo [skill_name_fr = "+skill_name_fr+", description_fr = "+description_fr+", skill_name_en = "+skill_name_en+", skill_id = "+skill_id+"]";
    }
}

File: app/src/main/java/com/example/zombicide/models/Skill.kt
package com.example.zombicide.models

import com.google.gson.annotations.SerializedName
import java.io.Serializable

data class Skill (

    @SerializedName("skill_id") val skill_id : Int,
    @SerializedName("skill_name") val skill_name : String,
    @SerializedName("skill_name_en") val skill_name_en : String,
    @SerializedName("description") val description : String,
    @SerializedName("description_en") val description_en : String
) : Serializable